docs/How-to/Run-a-full-Terra-node/Configure-general-settings.md

```diff
@@ -81,8 +74,41 @@ Once restarted, the LCD will be available.
 
 For more information about the Terra REST API endpoints, see the [Swagger documentation](https://lcd.terra.dev/swagger/).
 
-## Rosetta
+## Additional settings
+
+### `seed_mode`
+
+In seed mode, a node continuously crawls the network for peers, and upon incoming connection shares some peers and disconnects.
+
+```toml
+# Seed mode, in which node constantly crawls the network and looks for
+# peers. If another node asks it for addresses, it responds and disconnects.
+# Does not work if the peer-exchange reactor is disabled.
+seed_mode = false
+```
+
+### `seeds`
+
+When more peers are needed, the seed nodes you specify are dialed and a list of peers is returned. If enough peers are already in the address book, the `seeds` setting might never be used.
```

Comment:
Replace with: If you want to manually identify seed nodes, you may do so by editing this setting in the config.toml